### PR TITLE
qemu.tests: Add macvtap_event_notification new test case

### PR DESCRIPTION
**Pull request by FengYang**
Signed-off-by: Feng Yang <fyang@redhat.com>

### CODE REVIEW

File: qemu/tests/macvtap_event_notification.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    send_cmd(event_cmd, event_cmd_type)
+
+    error.context("Try to get qmp events in %s seconds!" % timeout,
+                  logging.info)
+    end_time = time.time() + timeout
+    qmp_monitors = vm.get_monitors_by_type("qmp")
+    qmp_num = len(qmp_monitors)
+    while time.time() < end_time:
+        for monitor in qmp_monitors:
+            event = monitor.get_event(event_check)
```
Comment (FengYang) [inline qemu/tests/macvtap_event_notification.py]: From the log, did not get NIC_RX_FILTER_CHANGED during guest bootup.

Only get event after ifconfig eth3 promisc.

Comment (FengYang) [inline qemu/tests/macvtap_event_notification.py]: Hi @suqinhuang.  Checked with qiguo.  This event may only find after boot or shutdown now.
It is not a key point. We have a check is enough. He will update the case.

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by ypu** [comment]
Thanks @FengYang and @suqinhuang will merge it.

**Pull request merged**